src/backend/marsha/core/views.py

```diff
@@ -342,6 +342,9 @@ def _get_base_app_data(self):
                 "uploadPollInterval": settings.FRONT_UPLOAD_POLL_INTERVAL,
                 # front is expecting duration in milliseconds
                 "attendanceDelay": settings.ATTENDANCE_PUSH_DELAY * 1000,
+                # In certain context, such as embedding the resource in content,
+                # we want to collapse the dashboard.
+                "dashboardCollapsed": "custom_embedded_resource" in self.request.POST,
```

Comment:
you should not also verify the value ?

Reply:
As you want, I have no opinion about this :sweat_smile:

Reply:
ok we can use it like this